src/content/docs/en/reference/api-reference.mdx

```diff
@@ -826,6 +826,200 @@ If true, the cookie is only set on https sites.
 
 Allows customizing how the cookie is serialized.
 
+### `session`
+
+<p>
+
+**Type:** `AstroSession`
+
+<Since v="5.1.0" />
+
+</p>
+
+`session` is an object that allows data to be stored between requests for [routes rendered on demand](/en/guides/on-demand-rendering/). It is associated with a cookie that contains the session ID, but differs from [`cookies`](#cookies) because the data itself is not stored in the cookie. This means there are no limits on the amount of data that can be stored in the session, and it can be trusted with secure data. The session is created when first used, and the session cookie is automatically set. The `session` object is `undefined` if no session storage has been configured, or if the current route is prerendered, and will log an error if you try to use it.
```

Comment:
```suggestion
`session` is an object that allows data to be stored between requests for [routes rendered on demand](/en/guides/on-demand-rendering/). 

It is associated with a cookie that contains the session ID only: the data itself is not stored in the cookie. This provides some key advantages over using [`cookies`](#cookies) for data storage: there are no limits on the amount of data that can be stored in the session, and it can be trusted with secure data.

The session is created when first used, and the session cookie is automatically set. The `session` object is `undefined` if no session storage has been configured, or if the current route is prerendered, and will log an error if you try to use it.
```